.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: ruby
+sudo: false
 rvm:
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.2.2
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
+  - 2.4.0
```

Comment:
isn't travis building against latest patch version when you do not specify them ?
By specifying ourselves, we need to update all our repo when a new patch is released.